docs/source/examples/ipc.md

```diff
@@ -4,7 +4,7 @@ This is an example demonstrating how you can enable the plasma shared memory sto
 
 Mosec's multi-stage pipeline requires the output data from the previous stage to be transferred to the next stage across python processes. This is coordinated via Unix domain socket between every Python worker process from all stages and the Rust controller process.
 
-By default, we serialize the data and directly transfer the bytes over the socket. However, users may find wrapping this IPC useful or more efficient for specific use cases. Therefore, we provide the `mosec.plugins.IPCWrapper` interface and an example implementation `PlasmaShmWrapper` based on [`pyarrow.plasma`](https://arrow.apache.org/docs/python/plasma.html).
+By default, we serialize the data and directly transfer the bytes over the socket. However, users may find wrapping this IPC useful or more efficient for specific use cases. Therefore, we provide the `mosec.plugins.IPCWrapper` interface and example implementation `PlasmaShmWrapper` based on [`pyarrow.plasma`](https://arrow.apache.org/docs/python/plasma.html) and `RedisShmWrapper` based on [`redis`](https://pypi.org/project/redis). We recommand using `RedisShmWrapper` for better performance.
```

Comment:
Is "better performance" verified experimentally?

Reply:
Yes, based on the benchmark experiment at https://github.com/mosecorg/numbin/blob/master/benchmark/bench.py, the results show that when comparing the performance ,Redis connected via a Unix socket > Plasma > Redis connected via a TCP socket.

Reply:
Cool! 

But the benchmark file mentioned above only tests the serialization / de-serialization performance, so you modify it to test the shared memory performance?

Reply:
Since it's a repo of serde library, I reused the cases while did not update it.

Reply:
I think the code and benchmark data can be uploaded to [mosecorg/benchmark](https://github.com/mosecorg/benchmark). We can add the link here.